_data/conferences.yml

```diff
@@ -2290,3 +2290,11 @@
   url: https://helvetic-ruby.ch
   twitter: helvetic_ruby
   mastodon: https://ruby.social/@helvetic_ruby
+
+- name: RubyConf 2023
```

Comment:
Should this go before Helvetic Ruby (looks like it should be in chronological order)

Reply:
Yes please!

Reply:
Yeah, it needs to be in chronological order in the YAML file itself